waf.go

```diff
@@ -165,7 +163,7 @@ func (r *Result) HasActions() bool {
 // Otherwise, it returns false along with an error detailing why.
 func SupportsTarget() (bool, error) {
 	wafSupportErrors := support.WafSupportErrors()
-	return len(wafSupportErrors) == 0, multierror.Append(nil, wafSupportErrors...).ErrorOrNil()
+	return wafSupportErrors == nil, errors.Join(wafSupportErrors...)
```

Comment:
I feel like these are making a somewhat dodgy assumption that `WafSupportErrors()` will return a nil slice if there is no error, when an empty slice would be semantically equivalent...

```suggestion
	err := errors.Join(support.WafSupportErrors())
	return err == nil, err
```

Reply:
errors.Join will return nil in both cases